norduniclient/models.py

```diff
@@ -511,6 +530,14 @@ def get_responsible_for(self):
             """
         return self._basic_read_query_to_dict(q)
 
+    def set_parent(self, org_handle_id):
+        q = """
+            MATCH (n:Node:Organization {handle_id: {handle_id}}), (m:Node:Organization {handle_id: {org_handle_id}})
```

Comment:
If you use the label Organization this method would need to be placed in an OrganizationModel class.